src/frontend/src/flows/manage/index.ts

```diff
@@ -200,12 +205,14 @@ export const renderManage = async (
 
 export const displayManagePage = renderPage(displayManageTemplate);
 
-export const displayManage = (
+export const displayManage = async (
   userNumber: bigint,
   connection: AuthenticatedConnection,
   devices_: DeviceData[]
-): Promise<void | AuthenticatedConnection> =>
-  new Promise((resolve) => {
+): Promise<void | AuthenticatedConnection> => {
+  // Fetch the dapps used in the teaser & explorer
+  const dapps = await getDapps();
```

Comment:
shouldn't the dapps list be loaded within `dappsExplorer` or any reason to load these that way?

Reply:
They're shown in the teaser (the marquee) on this page (which is a button that takes you to the explorer) but good catch